deploy/helm/metrics-discovery-config.yaml

```diff
@@ -98,74 +89,21 @@ processors:
 
   batch/metrics:
 {{ toYaml .Values.aws_fargate.metrics.autodiscovery.batch | indent 4 }}
-  
+
 {{- include "common-config.filter-remove-temporary-metrics" . | nindent 2 }}
 
   filter/histograms:
     metrics:
       metric:
         - 'type == METRIC_DATA_TYPE_HISTOGRAM and not(name == "k8s.rest_client_request_duration_seconds" or name == "k8s.workqueue_queue_duration_seconds")'
 
-{{- if .Values.otel.metrics.autodiscovery.prometheusEndpoints.filter }}
-  filter/metrics-discovery:
-    metrics:
-{{ toYaml .Values.otel.metrics.autodiscovery.prometheusEndpoints.filter | indent 6 }}
-{{- end }}
-
-{{- if .Values.otel.metrics.autodiscovery.prometheusEndpoints.customTransformations.counterToRate }}
-  cumulativetodelta/discovery:
-    include:
-      metrics:
-{{- range .Values.otel.metrics.autodiscovery.prometheusEndpoints.customTransformations.counterToRate }}
-        - {{ . }}
-{{- end }}
-      match_type: strict
-  deltatorate/discovery:
-    metrics:
-{{- range .Values.otel.metrics.autodiscovery.prometheusEndpoints.customTransformations.counterToRate }}
-      - {{ . }}
-{{- end }}
-{{- end }}
-
-{{- if ne .Values.otel.metrics.autodiscovery.prefix "k8s." }}
-  # in case the prefix differs from "k8s." we need to copy the required metrics
-  # so that SWO built-in dashboards works correctly
-{{- $arrayOfRequiredMetrics := list 
-  "etcd_disk_backend_commit_duration_seconds"
-  "etcd_disk_wal_fsync_duration_seconds" 
-  "etcd_network_client_grpc_received_bytes_total" 
-  "etcd_network_client_grpc_sent_bytes_total" 
-  "etcd_network_peer_received_bytes_total" 
-  "etcd_network_peer_sent_bytes_total" 
-  "etcd_server_leader_changes_seen_total" 
-  "etcd_server_proposals_applied_total" 
-  "etcd_server_proposals_committed_total"
-  "etcd_server_proposals_failed_total"
-  "etcd_server_proposals_pending"
-  "etcd_server_has_leader"
-  "etcd_mvcc_db_total_size_in_bytes"
-  "process_resident_memory_bytes"
-  "grpc_server_started_total"
-  "grpc_server_handled_total"
-  "rest_client_request_duration_seconds"
-  "rest_client_requests_total"
-  "workqueue_adds_total"
-  "workqueue_depth"
-  "workqueue_queue_duration_seconds"
-}}
-  metricstransform/copy-required-metrics:
-    transforms:
-    {{- $root := . }}
-    {{- range $index, $metric := $arrayOfRequiredMetrics }}
-      - include: {{ $root.Values.otel.metrics.autodiscovery.prefix }}{{ $metric }}
-        action: insert
-        new_name: k8s.{{ $metric }}
-    {{- end }}
-{{- end }}
+{{ include "common-discovery-config.processors" . | indent 2 }}
```

Comment:
This template includes processors for Istio metrics. Do we get them also in Fargate environment?

Reply:
Istio is probably not supported on Fargate, but if it is mixed cluster the istio can run on standard nodes so it makes sense to process that.